src/libPMacc/include/cuSTL/algorithm/mpi/Gather.tpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013, 2015 Heiko Burau, Benjamin Worpitz
+ * Copyright 2013 - 2015 Heiko Burau, Benjamin Worpitz, Alexander Grund
```

Comment:
Please change to `2013-2015` like in other files. 
Don't ask me why there are no spaces around^^